# Урок 3. Data Science: разработке и тестировании лекарств

## Домашнее задание.

> В онлайн-инструментах (и не только) используются данные о химических веществах в формате SMILES. Обычно он используется для описания структуры химических молекул. SMILES - это аббревиатура от Simplified Molecular Input Line Entry Specification.
Все основные данные о веществах можно найти в базе данных PubChem. PubChem в основном содержит небольшие молекулы, но также и более крупные молекулы, такие как нуклеотиды, углеводы, липиды, пептиды и химически модифицированные макромолекулы. Также есть информация о химических структурах, идентификаторах, химических и физических свойствах, биологической активности, патентах, данных о здоровье, безопасности, токсичности и многом другом.
Некоторые инструменты для предсказания мишеней:
> - **SwissTargetPrediction** - позволяет оценить наиболее вероятные макромолекулярные мишени небольшой молекулы, предполагаемой как биологически активная. Предсказание основано на сочетании двухмерного и трехмерного сходства с библиотекой из 370 000 известных активных веществ для более чем 3000 белков трех разных видов.
> - **Super-PRED** - веб-сервер прогнозирования соединений, основанный на модели машинного обучения с использованием логистической регрессии.
>- **Tools4miRs** - представляет собой веб-мета-сервер прогнозирования целей, включающий определяемые пользователем методы прогнозирования целей при анализе предоставленных пользователями данных.

### Задание:  
Найдите в интернете лекарство, вышедшее из употребления, сделайте поиск его структуры по химическим базам данных. Затем с помощью одного из описанных выше тулов предскажите мишени для этого лекарства. Изучите найденные мишени и предположите, какие заболевания может лечить данное лекарство при взаимодействии с предсказанными мишенями.


In [2]:
import pandas as pd
import numpy as np
import os
# import tarfile
import pprint
import seaborn as sns 
import matplotlib.pyplot as plt
%matplotlib inline

In [3]:
!chcp 65001

Active code page: 65001


In [4]:
pd.set_option('display.max_rows', None)

**Для изучения инструментов драг-дизайна возьмем лекарственное вещество _Фузафунгин_ (ТМ Биопарокс$^{®}$ (Bioparox$^{®}$), ЛАБОРАТОРИИ СЕРВЬЕ, ФРАНЦИЯ, РУ № П N015629/01). Полипептидный антибиотик для местного применения. Выделен из культуры гриба Fusarium lateritium (штамм 437). Использовался для лечения бактериальных инфекций верхних дыхательных путей. На текущий момент является лекарством с недоказанной эффективностью. Применение запрещено по всему миру**  
  
**Брутто формула: C$_{33}$H$_{57}$N$_3$O$_9$**

![Структура молекулы Фузафунгина](20_3_files/img/Fusafungine.png "Структура молекулы Фузафунгина")  


**В формате SMILES Фузафунгин записан так:**  
CC(C)C1C(=O)OC(C(=O)N(C(C(=O)OC(C(=O)N(C(C(=O)OC(C(=O)N1C)C(C)C)C(C)C)C)C(C)C)C(C)C)C)C(C)C



### Результаты SwissTargetPrediction  
Инструмент выдал 100 возможных мишеней для указанной структуры. У всех мишеней вероятность около `0,05`, что само по себе очень мало. Тем не менее рассмотрим результаты.

In [5]:
os.listdir('20_3_files')

['Fusafungine.png',
 'Indications.csv',
 'PI3K-Akt-signaling-pathway-picture.webp',
 'PI3K-р110_inhibitors.png',
 'PIK3CA_mutation_dispersion.jpg',
 'SwissTargetPrediction.csv',
 'Targets.csv']

In [6]:
stp = pd.read_csv('20_3_files/SwissTargetPrediction.csv', sep=',')
stp

Target             Common name  \
0                            FK506-binding protein 1A                  FKBP1A   
1                                    P-glycoprotein 1                   ABCB1   
2                    Proteasome Macropain subunit MB1                   PSMB5   
3                          Cholecystokinin A receptor                   CCKAR   
4                            Vasopressin V1a receptor                  AVPR1A   
5        Sodium channel protein type IX alpha subunit                   SCN9A   
6                       C-C chemokine receptor type 8                    CCR8   
7                                           Calpain 1            CAPN1 CAPNS1   
8          Cholecystokinin B receptor   (by homology)                   CCKBR   
9                   Thrombin and coagulation factor X                     F10   
10                        Arachidonate 5-lipoxygenase                   ALOX5   
11                   Lysosomal Pro-X carboxypeptidase                    PRCP   
12                              Adenosine A1 receptor                  ADORA1   
13                             Adenosine A2a receptor                 ADORA2A   
14                    Tyrosine-protein kinase ITK/TSK                     ITK   
15           Serine/threonine-protein kinase Aurora-A                   AURKA   
16                       Tyrosine-protein kinase JAK3                    JAK3   
17                       Tyrosine-protein kinase JAK1                    JAK1   
18                       Tyrosine-protein kinase JAK2                    JAK2   
19                                      DGAT1 protein                   DGAT1   
20                              Phosphodiesterase 10A                  PDE10A   
21                             Kinesin-like protein 1                   KIF11   
22                        Tyrosine-protein kinase SYK                     SYK   
23                             Bradykinin B2 receptor                  BDKRB2   
24         Cyclin-dependent kinase 5/CDK5 activator 1             CDK5R1 CDK5   
25                Cyclin-dependent kinase 2/cyclin E1              CCNE1 CDK2   
26                  Oxytocin receptor   (by homology)                    OXTR   
27                Acyl-CoA desaturase   (by homology)                     SCD   
28                          Cyclin-dependent kinase 2                    CDK2   
29  Carnitine O-palmitoyltransferase 1, liver isoform                   CPT1A   
30      Macrophage colony stimulating factor receptor                   CSF1R   
31                        Tyrosine-protein kinase ABL                    ABL1   
32      Vascular endothelial growth factor receptor 1                    FLT1   
33       Platelet-derived growth factor receptor beta                  PDGFRB   
34                   Stem cell growth factor receptor                     KIT   
35      Vascular endothelial growth factor receptor 3                    FLT4   
36      Vascular endothelial growth factor receptor 2                     KDR   
37                Fibroblast growth factor receptor 1                   FGFR1   
38                      Tyrosine-protein kinase TIE-2                     TEK   
39                                  Orexin receptor 2                  HCRTR2   
40                          p53-binding protein Mdm-2                    MDM2   
41                                  Orexin receptor 1                  HCRTR1   
42                            Focal adhesion kinase 1                    PTK2   
43               Translocator protein   (by homology)                    TSPO   
44            Vasopressin V2 receptor   (by homology)                   AVPR2   
45                            Prostanoid EP1 receptor                  PTGER1   
46                Cytochrome P450 1A1   (by homology)                  CYP1A1   
47                          Purinergic receptor P2Y12                  P2RY12   
48        Isocitrate dehydrogenase [NADP] cytoplasmic                    IDH1 

In [7]:
stp.describe()

Probability*
count  1.000000e+02
mean   5.564795e-02
std    2.789540e-17
min    5.564795e-02
25%    5.564795e-02
50%    5.564795e-02
75%    5.564795e-02
max    5.564795e-02

Вероятность взаимодействия со всеми мишенями одинакова, посмотрим уникальные значения в колонке `'Target'`

In [8]:
pd.set_option('display.max_rows', None)

In [9]:
stp['Target'].value_counts()

Target
FK506-binding protein 1A                                     1
Smoothened homolog   (by homology)                           1
Cyclin-dependent kinase 2/cyclin A                           1
Cyclin-dependent kinase 1/cyclin B                           1
Sodium channel protein type II alpha subunit                 1
Matrix metalloproteinase 14                                  1
Sodium channel protein type V alpha subunit                  1
MAP kinase ERK2                                              1
JAK2/JAK1                                                    1
JAK3/JAK1                                                    1
Phosphodiesterase 11A                                        1
Phosphodiesterase 5A                                         1
Neuropeptide Y receptor type 1                               1
P-glycoprotein 1                                             1
Histone deacetylase 1                                        1
Histone deacetylase 6                           

Все значения в колонке `'Target'` уникальны. Нет возможности выделить какую либо мишень в качестве приоритетеной.
Возможно получится получить информациюи из колонки `'Target Class'`:

In [10]:
stp['Target Class'].value_counts()

Target Class
Family A G protein-coupled receptor           24
Kinase                                        24
Enzyme                                         8
Protease                                       7
Voltage-gated ion channel                      6
Phosphodiesterase                              4
Reader                                         4
Nuclear receptor                               3
Other cytosolic protein                        3
Eraser                                         2
Oxidoreductase                                 2
Membrane receptor                              2
Other ion channel                              1
Family C G protein-coupled receptor            1
Family B G protein-coupled receptor            1
Frizzled family G protein-coupled receptor     1
Isomerase                                      1
Secreted protein                               1
Ligase                                         1
Cytochrome P450                                1
Primary

Наиболее часто встречается класс мишеней G-белок связанных рецепторов семейства А (GPCR), а также Киназы. Офильтруем значения по этим классам.

In [11]:
stp.sort_values(by='Target Class')

Target             Common name  \
46                Cytochrome P450 1A1   (by homology)                  CYP1A1   
27                Acyl-CoA desaturase   (by homology)                     SCD   
48        Isocitrate dehydrogenase [NADP] cytoplasmic                    IDH1   
53                      PI3-kinase p110-delta subunit                  PIK3CD   
29  Carnitine O-palmitoyltransferase 1, liver isoform                   CPT1A   
54                       PI3-kinase p110-beta subunit                  PIK3CB   
74                             SUMO-activating enzyme               SAE1 UBA2   
55                      PI3-kinase p110-gamma subunit                  PIK3CG   
58                       Arachidonate 15-lipoxygenase                  ALOX15   
61                              Histone deacetylase 1                   HDAC1   
60                              Histone deacetylase 6                   HDAC6   
59                             Cannabinoid receptor 1                    CNR1   
23                             Bradykinin B2 receptor                  BDKRB2   
99     Neuropeptide Y receptor type 5   (by homology)                   NPY5R   
47                          Purinergic receptor P2Y12                  P2RY12   
45                            Prostanoid EP1 receptor                  PTGER1   
44            Vasopressin V2 receptor   (by homology)                   AVPR2   
26                  Oxytocin receptor   (by homology)                    OXTR   
62                     Neuropeptide Y receptor type 1                   NPY1R   
39                                  Orexin receptor 2                  HCRTR2   
75                              Adenosine A3 receptor                  ADORA3   
92                    Proteinase-activated receptor 2                   F2RL1   
89             Sphingosine 1-phosphate receptor Edg-3                   S1PR3   
3                          Cholecystokinin A receptor                   CCKAR   
4                            Vasopressin V1a receptor                  AVPR1A   
6                       C-C chemokine receptor type 8                    CCR8   
87                              Neurokinin 3 receptor                   TACR3   
41                                  Orexin receptor 1                  HCRTR1   
8          Cholecystokinin B receptor   (by homology)                   CCKBR   
80                     Type-1 angiotensin II receptor                   AGTR1   
78                           Vasopressin V1b receptor                  AVPR1B   
12                              Adenosine A1 receptor                  ADORA1   
13                             Adenosine A2a receptor                 ADORA2A   
76                             Bradykinin B1 receptor                  BDKRB1   
86           Melanin-concentrating hormone receptor 1                   MCHR1   
77  Corticotropin releasing factor receptor 1   (b...                   CRHR1   
88  Metabotropic glutamate receptor 5   (by homology)                    GRM5   
63                 Smoothened homolog   (by homology)                     SMO   
0                            FK506-binding protein 1A                  FKBP1A   
37                Fibroblast growth factor receptor 1                   FGFR1   
67                                          JAK2/JAK1               JAK1 JAK2   
22                        Tyrosine-protein kinase SYK                     SYK   
24         Cyclin-dependent kinase 5/CDK5 activator 1             CDK5R1 CDK5   
68                                    MAP kinase ERK2                   MAPK1   
38                      Tyrosine-protein kinase TIE-2                     TEK   
25                Cyclin-dependent kinase 2/cyclin E1              CCNE1 CDK2   
28                          Cyclin-dependent kinase 2                    CDK2   
30      Macrophage colony stimulating factor receptor                   CSF1R   
17                       Tyrosine-protein kinase JAK1                    JAK1 

Так как с GPCR связывается достаточно большое количество регуляторных агентов, возможно для дальнейшего использования потребуется некоторая модификация молекулы. Либо использование ее в качестве субстрата/транспорта. 

### Результаты Super-PRED  
Инструмент выдает мишени, а также заболевания, связанные с мишенями.

In [12]:
sp_trg = pd.read_csv('20_3_files/Targets.csv', sep=',')
sp_trg

Target Name      ChEMBL-ID  \
0                             Cannabinoid CB2 receptor      CHEMBL253   
1                                          Cathepsin D     CHEMBL2581   
2                        G-protein coupled receptor 55  CHEMBL1075322   
3               Nuclear factor NF-kappa-B p105 subunit     CHEMBL3251   
4            Transcription intermediary factor 1-alpha  CHEMBL3108638   
5                               Bloom syndrome protein  CHEMBL1293237   
6                      Tyrosyl-DNA phosphodiesterase 1  CHEMBL1075138   
7          Acyl coenzyme A:cholesterol acyltransferase     CHEMBL2265   
8                                        Cyclophilin A     CHEMBL1949   
9                       Heat shock protein HSP 90-beta     CHEMBL4303   
10                    Glycine receptor subunit alpha-1     CHEMBL5845   
11                         Casein kinase II alpha/beta  CHEMBL3038477   
12                                Cytochrome P450 19A1     CHEMBL1978   
13                              Histone deacetylase 10     CHEMBL5103   
14                                    Aminopeptidase N     CHEMBL1907   
15              G-protein coupled bile acid receptor 1     CHEMBL5409   
16                           Formyl peptide receptor 1     CHEMBL3359   
17                    Hypoxia-inducible factor 1 alpha     CHEMBL4261   
18                                         Cathepsin B     CHEMBL4072   
19                          Matrix metalloproteinase 9      CHEMBL321   
20                                 Pregnane X receptor     CHEMBL3401   
21                             Integrin alpha-V/beta-3  CHEMBL1907598   
22   Voltage-gated N-type calcium channel alpha-1B ...     CHEMBL4478   
23                          Matrix metalloproteinase-2      CHEMBL333   
24                    Lysosomal Pro-X carboxypeptidase     CHEMBL2335   
25                                 DCN1-like protein 1  CHEMBL4105838   
26                          DNA topoisomerase II alpha     CHEMBL1806   
27                       PI3-kinase p110-delta subunit     CHEMBL3130   
28                       G-protein coupled receptor 35  CHEMBL1293267   
29                              Adenosine A2b receptor      CHEMBL255   
30                         NT-3 growth factor receptor     CHEMBL5608   
31   Cystic fibrosis transmembrane conductance regu...     CHEMBL4051   
32                                Toll-like receptor 4     CHEMBL5255   
33                     L-lactate dehydrogenase A chain     CHEMBL4835   
34                          Tryptophan 2,3-dioxygenase     CHEMBL2140   
35                          Matrix metalloproteinase 8     CHEMBL4588   
36              Sphingosine 1-phosphate receptor Edg-8     CHEMBL2274   
37                Serine/threonine-protein kinase mTOR     CHEMBL2842   
38                Dual specificity protein kinase CLK4     CHEMBL4203   
39                 Ribosomal protein S6 kinase alpha 1     CHEMBL2553   
40                          Matrix metalloproteinase-1      CHEMBL332   
41         Nuclear factor erythroid 2-related factor 2  CHEMBL1075094   
42                     Transmembrane protease serine 6  CHEMBL1795139   
43                       C-C chemokine receptor type 1     CHEMBL2413   
44          Cyclin-dependent kinase 5/CDK5 activator 1  CHEMBL1907600   
45   Voltage-gated T-type calcium channel alpha-1H ...     CHEMBL1859   
46                    Cysteinyl leukotriene receptor 2     CHEMBL4330   
47                               Phosphodiesterase 11A     CHEMBL2717   
48   Dual specificity mitogen-activated protein kin...     CHEMBL2964   
49                 T-cell protein-tyrosine phosphatase     CHEMBL3807   
50       Sodium channel protein type III alpha subunit     CHEMBL5163   
51                      Serotonin 3a (5-HT3a) receptor     CHEMBL1899   
52                      Proteasome subunit beta type-9  CHEMBL1944495   
53              Tyrosine-protein kinase receptor TYRO3     CHEMBL5314   
54           Multidrug resis

In [13]:
sp_trg.describe()

Target Name  ChEMBL-ID UniProt ID PDB Visualization  \
count                        127        127        127               127   
unique                       127        127        126               104   
top     Cannabinoid CB2 receptor  CHEMBL253     Q00535     Not Available   
freq                           1          1          2                24   

               TTD ID Probability Model accuracy  
count             127         127            127  
unique             68         123            100  
top     Not Available       57.7%           100%  
freq               58           2              8

В качестве наиболее вероятной (**Probability**=95,73%; **Model accuracy**=98,95%) мишени Super-PRED выдает каннабиоидный рецептор 2-го типа (CB$_2$), что в теории позволяет рассматривать Фузафунгин в качестве возможного антидепрессанта.

In [14]:
sp_ind = pd.read_csv('20_3_files/Indications.csv', sep=',')
sp_ind

Target Name ChEMBL-ID  \
0                                          Cathepsin D    T67102   
1                                          Cathepsin D    T67102   
2                        G-protein coupled receptor 55    T87670   
3          Acyl coenzyme A:cholesterol acyltransferase    T76369   
4          Acyl coenzyme A:cholesterol acyltransferase    T76369   
5          Acyl coenzyme A:cholesterol acyltransferase    T76369   
6          Acyl coenzyme A:cholesterol acyltransferase    T76369   
7          Acyl coenzyme A:cholesterol acyltransferase    T76369   
8          Acyl coenzyme A:cholesterol acyltransferase    T76369   
9          Acyl coenzyme A:cholesterol acyltransferase    T76369   
10                                       Cyclophilin A    T47081   
11                    Glycine receptor subunit alpha-1    T50269   
12                         Casein kinase II alpha/beta    T51565   
13                         Casein kinase II alpha/beta    T51565   
14                                Cytochrome P450 19A1    T13260   
15                                Cytochrome P450 19A1    T13260   
16                                Cytochrome P450 19A1    T13260   
17                                Cytochrome P450 19A1    T13260   
18                                Cytochrome P450 19A1    T13260   
19                                Cytochrome P450 19A1    T13260   
20                                Cytochrome P450 19A1    T13260   
21                              Histone deacetylase 10    T94324   
22                                    Aminopeptidase N    T67272   
23              G-protein coupled bile acid receptor 1    T86273   
24              G-protein coupled bile acid receptor 1    T86273   
25                           Formyl peptide receptor 1    T87831   
26                           Formyl peptide receptor 1    T87831   
27                                         Cathepsin B    T61746   
28                                         Cathepsin B    T61746   
29                                         Cathepsin B    T61746   
30                                         Cathepsin B    T61746   
31                                         Cathepsin B    T61746   
32                                         Cathepsin B    T61746   
33                                         Cathepsin B    T61746   
34                                         Cathepsin B    T61746   
35                                         Cathepsin B    T61746   
36                                         Cathepsin B    T61746   
37                                         Cathepsin B    T61746   
38                                         Cathepsin B    T61746   
39                                         Cathepsin B    T61746   
40                                         Cathepsin B    T61746   
41                                         Cathepsin B    T61746   
42                                         Cathepsin B    T61746   
43                                         Cathepsin B    T61746   
44                                         Cathepsin B    T61746   
45                                         Cathepsin B    T61746   
46                                 Pregnane X receptor    T82702   
47                             Integrin alpha-V/beta-3    T67103   
48                             Integrin alpha-V/beta-3    T67103   
49   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
50   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
51   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
52   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
53   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
54   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
55   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
56   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
57   Voltage-gated N-type calcium channel alpha-1B ...    T38338   
58                          DNA

In [15]:
sp_ind.describe()

Target Name ChEMBL-ID  \
count                                       460       460   
unique                                       69        67   
top     Platelet-derived growth factor receptor    T53524   
freq                                         33        55   

                                     Indication Probability Model accuracy  
count                                       460         460            460  
unique                                      235          69             58  
top     Solid tumour/cancer [ICD-11: 2A00-2F9Z]      55.66%         71.67%  
freq                                         30          33             33

In [16]:
sp_ind['Target Name'].value_counts()

Target Name
Platelet-derived growth factor receptor                    33
Serine/threonine-protein kinase mTOR                       33
Androgen Receptor                                          29
Platelet-derived growth factor receptor alpha              22
Indoleamine 2,3-dioxygenase                                20
Cathepsin B                                                19
Neuronal acetylcholine receptor protein alpha-7 subunit    18
Ghrelin receptor                                           18
Cathepsin L                                                18
C-C chemokine receptor type 2                              17
PI3-kinase p110-delta subunit                              15
P2X purinoceptor 7                                         13
Adenosine A2b receptor                                     12
Cyclin-dependent kinase 1/cyclin B1                        11
Histone-lysine N-methyltransferase EZH2                    10
Voltage-gated N-type calcium channel alpha-1B subunit     

В модели "Показания предсказанных мишеней" наиболее часто (n=33 из 460) встречаются рецептор тромбоцитарного фактора роста (PDGFR) и серин-треониновая протеинкиназа мишень Рапамицина млекопитающих (mTOR). Оба белка участвуют во внутриклеточных сигнальных путях при различных видах рака. В качестве наиболее вероятной (Probability=91,51%%; Model accuracy=98,95%) указана мишень Катепсин D, относящаяся к лизосомальным аспартил протеазам.

### Tools4miRs  
Пока не разобрался как пользоваться этим инструментом. Предполагаю, что в него нужно вносить РНК интересующих мишеней. Поиска по веществу не нашел. Возможно не там искал...

__*Комментарии преподавателя:*__<br>
*Дарья Бородко・Преподаватель<br>*

>*Добрый день!  
Спасибо за подробное исследование таблички СвиссТаргет - там можно заметить, что лекарство взаимодействует с несколькими субъединицами PI3 киназы, которая входит в сигнальный каскад нескольких рецепторов. Соответственно было бы неплохо изучить, что это за белок и с какими болезнями ассоциирован.
Очень хорошая работа с суперпред :) Спасибо, было приятно почитать*

__*Ответ:*__<br>
В качестве возможных мишеней указаны ${\alpha, \beta}$ и ${\gamma}$ изоформы каталитической субъединицы `p110` фосфатидилинозитол-3-киназы (PI3K). PI3K является одним из центральных компонентов PI3K/AKT/mTOR внутриклеточного сигнального пути. [Это один из универсальных сигнальных путей, характерных для большинства клеток человека.](https://ru.wikipedia.org/wiki/%D0%A1%D0%B8%D0%B3%D0%BD%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9_%D0%BF%D1%83%D1%82%D1%8C_PI3K/AKT/mTOR "Статья про PI3K/AKT/mTOR путь в Википедии") Он отвечает за уход от апоптоза, рост, пролиферацию клеток, метаболизм. Также у этого сигнального пути есть несколько тканеспецифичных функций, например, в работе сердца.<br>

![PI3K/AKT/mTOR сигнальный путь](20_3_files/img/PI3K-Akt-signaling-pathway-picture.webp "PI3K/AKT/mTOR сигнальный путь")

PI3K состоит из каталитической субъединицы `p110`и регуляторной субъединицы `p55/p85`. PI3K кодируется геном PIK3CA. Мутации в 9 и 20 экзонах этого гена приводят дисрегуляции этого сигнального пути и встречаются в 40% случаев HR+/HER2- подтипа (люминальный А) рака молочной железы.
Экзон 20 кодирует каталитический домен р110${\alpha}$, и мутации в этом домене гена PIK3CA могут прямо стимулировать конститутивную ферментативную активность PI3K. Экзон 9 кодирует хеликазный домен р110${\alpha}$, и мутация в этом домене подавляют ингибирующее взаимодействие N-концевого участка р85 с р110${\alpha}$. Часть более редких мутаций обнаружена в домене, кодирующем N-концевой участок р110${\alpha}$, взаимодействующий с р85. И такой тип мутаций усиливает липидную активность PI3K, но не влияет на образование комплекса р85${\alpha}$ – р110${\alpha}$.

В опухолевых клетках РМЖ определен спектр наиболее характерных изменений в гене PIK3CA, определяющий патологическую активацию каскада PI3K-Akt-mTOR. Высокая частота миссенс-мутаций в 9 и 20 экзонах соответствует аминокислотам E542K и E545K в экзоне 9 хеликазного домена; в экзоне 20 киназного домена – H1047R и H1047L; наиболее частая мутация для люминального А подтипа РМЖ – E545K

![Наиболее частые мутации PIK3CA](20_3_files/img/PIK3CA_mutation_dispersion.jpg "Наиболее частые мутации PIK3CA")

Наличие мутации PIK3CA негативно влияет на прогноз течнаия РМЖ, в частности снижает эффективность проводимой у таких пациентов терапии ингибиторами циклин-зависимых киназ CDK4/6 (ATX L01EF: Рибоциклиб, Палбоциклиб, Абемациклиб) и mTOR-ингибиторами (АТХ L01EG: Эверолимус, Темсиролимус, Ридафоролимус), а также [определяет резистентность опухоли к эндокринной терапии.](http://www.cancergenome.ru/mutations/PIK3CA/ "Статья о PIK3CA мутации на cancergenome.ru")

Существуют селективные и неселективные изоформные ингибиторы PI3K-р110

![Ингибиторы PI3K-р110](20_3_files/img/PI3K-р110_inhibitors.png "Ингибиторы PI3K-р110")

Наиболее селективным является Алпелисиб, однако вследствие блокировки PI3K у пациентов во время терапии может развиваться гипергликемия, так как PI3K в том числе участвует в передаче сигнала от рецептора инсулина.

Также в качестве вероятных мишеней указаны Янус ассоциированные тирозин-киназы `JAK1, JAK2` и  `JAK3`. Взаимодействие Фузафунгина с этими мишенями теоретически можно использовать в терапии некоторых гематологических и ревматологических заболеваний.

С помощью __*Open Targets Platform*__ можно оценить возможность применения для каждой из мишеней в конкретных заболеваниях. Например посмотрим ассоциации для `PI3-kinase p110-delta subunit`:

In [17]:
t = pd.read_table('20_3_files/ENSG00000171608-associated-targets.tsv', sep='\t')
t

name  \
0                        activated PI3K-delta syndrome   
1                         chronic lymphocytic leukemia   
2    Combined immunodeficiency with facio-oculo-ske...   
3                                  follicular lymphoma   
4    combined immunodeficiency with faciooculoskele...   
5                                             neoplasm   
6                           neoplasm of mature B-cells   
7                                non-Hodgkins lymphoma   
8                                             lymphoma   
9                        marginal zone B-cell lymphoma   
10                           combined immunodeficiency   
11                                    eosinophil count   
12                                      monocyte count   
13                                              cancer   
14                                   lymphoid neoplasm   
15                                       breast cancer   
16                                    lymphocyte count   
17                                      bronchiectasis   
18                 eosinophil percentage of leukocytes   
19                                Mantle cell lymphoma   
20                                  endometrial cancer   
21                               breast adenocarcinoma   
22               head and neck squamous cell carcinoma   
23                                   brain measurement   
24                          gut microbiome measurement   
25                           pancreatic adenocarcinoma   
26                   cortical surface area measurement   
27                                           infection   
28                                     leukocyte count   
29                   monocyte percentage of leukocytes   
30                          Testicular Germ Cell Tumor   
31                                 lung adenocarcinoma   
32                   head and neck malignant neoplasia   
33                                    genetic disorder   
34                          inflammatory bowel disease   
35                       diffuse large B-cell lymphoma   
36                             glioblastoma multiforme   
37                              gastric adenocarcinoma   
38                                   Hodgkins lymphoma   
39                       non-small cell lung carcinoma   
40                                            melanoma   
41                        acute lymphoblastic leukemia   
42                       Waldenstrom macroglobulinemia   
43                                    multiple myeloma   
44                                     medulloblastoma   
45                                     prostate cancer   
46                                       neuroblastoma   
47              unspecified peripheral T-cell lymphoma   
48                    common variable immunodeficiency   
49                        metastatic colorectal cancer   
50                                     breast neoplasm   
51                                      thyroid cancer   
52                        squamous cell lung carcinoma   
53                                endometrial neoplasm   
54                       metastatic malignant neoplasm   
55                    pancreatic ductal adenocarcinoma   
56                            classic Hodgkin lymphoma   
57                                   esophageal cancer   
58                         isolated agammaglobulinemia   
59                              acute myeloid leukemia   
60      combined immunodeficiency with skin granulomas   
61                     Central Nervous System Lymphoma   
62                                       BENTA disease   
63                     pancreatic neuroendocrine tumor   
64                                            leukemia   
65                                 immunodeficiency 62   
66   severe combined immunodeficiency due to CTPS1 ...   
67   immunodeficiency 73c with defective neutrophil...   
68                         myeloproliferative di

__*Комментарии преподавателя:*__<br>
*Дарья Бородко・Преподаватель<br>*

>Добрый день!<br>
Спасибо за подробное исследование таблички СвиссТаргет - там можно заметить, что лекарство взаимодействует с несколькими субъединицами PI3 киназы, которая входит в сигнальный каскад нескольких рецепторов. Соответственно было бы неплохо изучить, что это за белок и с какими болезнями ассоциирован.  
Очень хорошая работа с суперпред :) Спасибо, было приятно почитать

~__**Исправлено**__~